src/device.rs

```diff
@@ -343,13 +357,15 @@ pub trait DeviceV1_0 {
 
     unsafe fn cmd_bind_vertex_buffers(&self,
                                       command_buffer: vk::CommandBuffer,
+                                      first_binding: vk::uint32_t,
                                       buffers: &[vk::Buffer],
-                                      offsets: &vk::DeviceSize) {
+                                      offsets: &[vk::DeviceSize]) {
+        debug_assert_eq!(buffers.len(), offsets.len());
         self.fp_v1_0().cmd_bind_vertex_buffers(command_buffer,
-                                               0,
+                                               first_binding,
```

Comment:
This should be necessary imo if you only want to update a part of the vertex input bindings,
`first_binding` states the first vertex input binding offset, else we would have to set it to 0 and have to update all input bindings.

Corresponding specs: (https://www.khronos.org/registry/vulkan/specs/1.0/man/html/vkCmdBindVertexBuffers.html)

> The values taken from elements i of pBuffers and pOffsets replace the current state for the vertex input binding firstBinding + i, for i in [0, bindingCount).